misc/ref/table.py.out

```diff
@@ -424,14 +424,15 @@ SLB_2011_ZSB_2013.mg_perovskite                                     2.445e-05 2.
 SLB_2011_ZSB_2013.periclase                                         1.124e-05  1.61e+11      3.8      1.31e+11      2.1     0.0403 2   767.0         1.36  1.7     2.8
 SLB_2011_ZSB_2013.stishovite                                        1.402e-05  3.14e+11      3.8       2.2e+11      1.9     0.0601 3  1108.0         1.37  2.8     4.6
 SLB_2011_ZSB_2013.wuestite                                          1.226e-05  1.79e+11      4.9 59000000000.0      1.4     0.0718 2   454.0         1.53  1.7    -0.1
-Name                                                  V_0          K_0 Kprime_0           G_0 Gprime_0 molar_mass   n  Debye_0 grueneisen_0     q_0  eta_s_0
-other.Fe_Dewaele                                 6.75e-06    1.634e+11     5.38           nan      nan   0.055845   1                                       
```

Comment:
They will do now. I've restructured table.py so that it can output minerals with different eoses from the same file.